packages/react-reconciler/src/ReactFiberBeginWork.js

```diff
@@ -840,6 +841,68 @@ function mountLazyComponent(
   return child;
 }
 
+function mountIncompleteComponent(
+  _current,
+  workInProgress,
+  renderExpirationTime,
+) {
+  if (_current !== null) {
+    // An incomplete component only mounts if it suspended inside a non-
+    // concurrent tree, in an inconsistent state. We want to tree it like
+    // a new mount, even though an empty version of it already committed.
+    // Disconnect the alternate pointers.
+    _current.alternate = null;
+    workInProgress.alternate = null;
+    // Since this is conceptually a new fiber, schedule a Placement effect
+    workInProgress.effectTag |= Placement;
+  }
+
+  // Promote the fiber to a class and try rendering again.
+  workInProgress.tag = ClassComponent;
+  const Component = workInProgress.type;
+  const unresolvedProps = workInProgress.pendingProps;
+  const resolvedProps =
+    workInProgress.elementType === Component
```

Comment:
Any reason this is here instead of beginWork like the others?

Reply:
@sebmarkbage Nah, I'll change